libraries/AP_HAL_ChibiOS/hwdef/KakuteF7/hwdef.dat

```diff
@@ -108,11 +108,11 @@ PE7 UART7_RX UART7
 PE8 UART7_TX UART7
 
 # Motors
-PB1  TIM1_CH3N TIM1 PWM(1) GPIO(50)
```

Comment:
this is changing more than just the ordering, its also changing the timer mapping. Why is that being done?

Reply:
Not intentional, just matching the Betaflight map. Thought it was mostly arbitrary, seems you think otherwise. That's fine. However, when i change the timer map back it then doesn't map to the pins correctly.. so it will need a different timer change if we want a different pinout mapping

Reply:
if matching Betaflight is now the goal then core Ardupilot code needs to be reworked and Betaflight motor scheme and rotation of motors enforced as well. Then matching of the Betaflight map will start making sense. All i see now here is the intent to brake something that was actually working very well for most people. At lest move existing hwdef to the Kakute F7 AIO so people with that controller would not have to deal with series of trial and errors of what output is not supposed to be associated with what motor.